Agents/CEAAgent/cea-agent/src/main/resources/CEAAgentConfig.properties

```diff
@@ -11,16 +11,44 @@ uri.ontology.ontoems=https://www.theworldavatar.com/kg/ontoems/
 uri.ontology.geoliteral=http://www.bigdata.com/rdf/geospatial/literals/v1#
 uri.service.geo=http://www.bigdata.com/rdf/geospatial#
 
-# TheWorldAvatar Blazegraph is the default endpoint for queries, with the namespace mappings provided by the following targetResourceIDs
+#--Stack Access Agent Configuration--#
+# Stack access agent base URL
+# DO NOT CHANGE
```

Comment:
This change is done internally by the agent, where it will automatically get the stack name using StackClient and replace "<STACK NAME>" with the actual stack name without the user/developer needing to change it manually. If "<STACK NAME>" were to be changed, the string replacement will not happen.

Reply:
Ok. That's a rather important and non-obvious piece of information! Can you please add that to the comment?